analyzers/src/SonarAnalyzer.Common/Helpers/KnownType.cs

```diff
@@ -256,10 +257,14 @@ public sealed partial class KnownType
         public static readonly KnownType System_Data_Common_CommandTrees_DbExpression = new("System.Data.Common.CommandTrees.DbExpression");
         public static readonly KnownType System_Data_DataSet = new("System.Data.DataSet");
         public static readonly KnownType System_Data_DataTable = new("System.Data.DataTable");
+        public static readonly KnownType System_Data_Entity_Core_Objects_ObjectQuery = new("System.Data.Entity.Core.Objects.ObjectQuery");
         public static readonly KnownType System_Data_Entity_Database = new("System.Data.Entity.Database");
         public static readonly KnownType System_Data_Entity_DbSet = new("System.Data.Entity.DbSet");
         public static readonly KnownType System_Data_Entity_DbSet_TEntity = new("System.Data.Entity.DbSet", "TEntity");
+        public static readonly KnownType System_Data_Entity_Infrastructure_DbQuery = new("System.Data.Entity.Infrastructure.DbQuery");
+        public static readonly KnownType System_Data_Entity_Infrastructure_DbQuery_TResult = new("System.Data.Entity.Infrastructure.DbQuery", "TResult");
```

Comment:
SC: [Define a constant instead of using this literal 'TResult' 28 times.](https://sonarcloud.io/project/issues?pullRequest=8593&open=AY1BFwsc_mVrn-4sFgj7&id=sonaranalyzer-dotnet)

---

Can be resolved as won't fix, there similar issue in the file.